src/components/stake/StakeValidatorFilter.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+    <div class="validator-filter">
+        <div class="validator-filter-wrapper">
+            <button class="validator-switch" :class="state === FilterState.TRUST?'selected':''"
+                @click="ctrl.stateChange(FilterState.TRUST)">
+                {{ $t('TrustScore') }}
+            </button>
+            <button class="validator-switch" :class="state === FilterState.PAYOUT?'selected':''"
+                @click="ctrl.stateChange(FilterState.PAYOUT)">
+                {{ $t('Payout Time') }}
+            </button>
+            <button class="validator-switch" :class="state === FilterState.REWARD?'selected':''"
+                @click="ctrl.stateChange(FilterState.REWARD)">
+                {{ $t('Reward') }}
+            </button>
+            <span class="validator-search" @click="ctrl.stateChange(FilterState.SEARCH)">
+                <SearchIcon/>
+            </span>
```

Comment:
This should also be a `button` or something to be usable via keyboard navigation.
Additionally, it should probably have hover and focus styles.